pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -64,7 +64,7 @@ type ElasticsearchNode struct {
 }
 
 type ElasticsearchStorageSpec struct {
-	StorageClassName string             `json:"storageClassName,omitempty"`
+	StorageClassName *string            `json:"storageClassName,omitempty"`
```

Comment:
this is part of addressing https://bugzilla.redhat.com/show_bug.cgi?id=1689762
this may break CLO when we specify a storageClassName -- will create a follow up fix

Reply:
will need to do a `dep ensure -update` for this